qa/tasks/ceph_manager.py

```diff
@@ -260,7 +260,7 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
                 while proceed():
                     proc = exp_remote.run(args=cmd, wait=True,
                                           check_status=False,
-                                          stdout=BytesIO(), stderr=BytesIO())
+                                          stdout=StringIO(), stderr=StringIO())
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

Reply:
sure, updated!